czech/admin/config_smilies.lang.php

```diff
@@ -5,60 +5,60 @@
  *
  */
 
-$l['smilies'] = "Smilies";
-$l['manage_smilies'] = "Manage Smilies";
-$l['manage_smilies_desc'] = "This section allows you to edit and delete your smilies.";
-$l['add_smilie'] = "Add New Smilie";
-$l['add_smilie_desc'] = "Here you can add a single new smilie.";
-$l['add_multiple_smilies'] = "Add Multiple Smilies";
-$l['add_multiple_smilies_desc'] = "Here you can add multiple new smilies at once.";
-$l['edit_smilie'] = "Edit Smilie";
-$l['edit_smilie_desc'] = "Here you can edit a single smilie.";
-$l['mass_edit'] = "Mass Edit";
-$l['mass_edit_desc'] = "Here you can easily edit all your smilies in one go.";
+$l['smilies'] = "Smajlíci";
+$l['manage_smilies'] = "Správa emotikonů";
```

Comment:
Taky jsem nad tím přemýšlel. Ale OK. Nic proti tomu nemám. Jsem rád, že ses na to podíval.